ts/features/wallet/paypal/PaypalCard.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import * as pot from "italia-ts-commons/lib/pot";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
+import { connect } from "react-redux";
+import BaseCardComponent from "../component/card/BaseCardComponent";
+import paypalLogoExt from "../../../../img/wallet/payment-methods/paypal-logo.png";
+import paypalLogoMin from "../../../../img/wallet/cards-icons/paypal_card.png";
+import { Body } from "../../../components/core/typography/Body";
+import { BrandImage } from "../component/card/BrandImage";
+import { GlobalState } from "../../../store/reducers/types";
+import { getFavoriteWalletId } from "../../../store/reducers/wallet/wallets";
+import IconFont from "../../../components/ui/IconFont";
+import variables from "../../../theme/variables";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import { PayPalPaymentMethod } from "../../../types/pagopa";
+
+type Props = {
+  paypal: PayPalPaymentMethod;
+} & ReturnType<typeof mapStateToProps>;
+
+const styles = StyleSheet.create({
+  paypalLogoExt: {
+    width: 132,
+    height: 33,
+    resizeMode: "contain"
+  },
+  row: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  }
+});
+
+const topLeft = (isFavourite: boolean) => (
+  <View style={styles.row}>
+    <View style={IOStyles.flex}>
+      <Image source={paypalLogoExt} style={styles.paypalLogoExt} />
+    </View>
+    {isFavourite && (
+      <IconFont name={"io-filled-star"} color={variables.brandPrimary} />
+    )}
+  </View>
+);
+
+const PaypalCard: React.FunctionComponent<Props> = (props: Props) => (
+  <BaseCardComponent
+    topLeftCorner={topLeft(
+      pot.getOrElse(
+        pot.map(props.favoriteWalletId, id => props.paypal.idWallet === id),
+        false
+      )
+    )}
+    bottomLeftCorner={<Body>{props.paypal.info.emailPp}</Body>}
```

Comment:
```suggestion
    bottomLeftCorner={
      <Body style={[IOStyles.flex, { paddingRight: 16 }]} numberOfLines={1}>
        {props.paypal.info.emailPp}
      </Body>
    }
```
In order to avoid overflow:
<img width="418" alt="Schermata 2021-11-22 alle 11 06 53" src="https://user-images.githubusercontent.com/26501317/142842297-e5e89995-8bc1-452f-915b-ea9bbc1c2f19.png">

Reply:
🤣 
changed with e0b5f9ccc058974251928ca550710d7de4de4bfb